pkgs/servers/imaginary/default.nix

```diff
@@ -11,6 +17,14 @@ buildGoModule rec {
     hash = "sha256-oEkFoZMaNNJPMisqpIneeLK/sA23gaTWJ4nqtDHkrwA=";
   };
 
+  patches = [
```

Comment:
can we document what this patch does, why we are including it, and when we can remove it?